hack/build-go.sh

```diff
@@ -16,6 +16,10 @@
 
 # This script is a vestigial redirection.  Please do not add "real" logic.
 
+# This script runs `make all` command.
+# The command conpiles all Makefile configs.
```

Comment:
`conpiles` ? Spelling mistake ?

Reply:
Thank you for your review. I will fix it.